internal/handlers/common/params.go

```diff
@@ -258,3 +258,61 @@ func addNumbers(v1, v2 any) (any, error) {
 		return nil, errUnexpectedLeftOpType
 	}
 }
+
+// GetPositiveNumber returns doc's value for key or protocol error for missing or invalid parameter.
+func GetPositiveNumber(document *types.Document, key string) (int32, error) {
+	v, err := document.Get(key)
+	if err != nil {
+		return 0, nil
```

Comment:
> GetPositiveNumber returns doc's value for key or _protocol error for missing_ or invalid parameter.

Here we have a missing key but do not return a protocol error. Either description or code is wrong. I think we should return the error there. Do we have a test for that?

Reply:
Fixed, no we don't have a test

Reply:
Fixed

Reply:
fixed